default-plugins/status-bar/src/tip/cache.rs

```diff
@@ -0,0 +1,105 @@
+use std::fs::File;
+use std::io::{BufReader, BufWriter};
+use std::path::PathBuf;
+
+use serde::{Deserialize, Serialize};
+use serde_json::Error as SerdeError;
+
+pub const CACHE_FILE_PATH: &str = "/tmp/cache.json";
+pub const MAX_CACHED_COUNT: usize = 10;
+
+#[derive(Debug)]
+pub struct Cache {
+    pub path: PathBuf,
+    pub data: Vec<CachedTip>,
+}
+
+#[derive(Debug, Serialize, Deserialize, Eq, PartialEq, Hash, Clone)]
+pub struct CachedTip {
+    name: String,
+    hitted: usize,
```

Comment:
I think `hit`, or `hits` would be cleaner here instead of `hitted`.

Reply:
Thank you :) beside this.. there must be many strange name. so i will check all the code again!